sphinx/config.py

```diff
@@ -32,6 +32,7 @@
 CONFIG_TYPE_WARNING = "The config value `{name}' has type `{current.__name__}', " \
                       "defaults to `{default.__name__}.'"
 
+CONFIG_EXTENSION_BLACKLIST = {"sphinxjp.themecore": "1.2"}
```

Comment:
It seems the value means a version the extension merged. But it is hard to understand from the definition.
It would be better if there is comment about its structure.